mta-sts.md

```diff
@@ -302,7 +302,7 @@ sender fetches a HTTPS resource (policy) from a host at `policy.mta-sts` in the
 Policy Domain. The policy is served from a "well known" URI:
 `https://policy.mta-sts.example.com/.well-known/mta-sts/current`. To consider 
```

Comment:
What do you mean? For example.com, it would be 

https://policy.mta-sts.example.com/.well-known/mta-sts/current

(Separate to this PR, as discussed on the UTA list and on https://github.com/mrisher/smtp-sts/issues/60, we may want to get rid of the nesting to make wildcard certs work here.)